scripts/sanitise-db.sql

```diff
@@ -339,9 +361,17 @@ DO $$
 BEGIN
     PERFORM anonymize_with_lorem_ipsum('funding', 'data_blob', 'funding_source');   --- LARGE TABLE
     PERFORM anonymize_with_lorem_ipsum('funding_comment', 'data_blob', 'comment');
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'question');
+    PERFORM anonymize_with_lorem_ipsum('funding_question', 'data_blob', 'indicator');
     PERFORM anonymize_with_lorem_ipsum('private_investment', 'data_blob', 'additional_comments');
     PERFORM anonymize_with_lorem_ipsum('programme_progress', 'data_blob', 'answer');
 
+    PERFORM anonymize_with_lorem_ipsum('project_progress', 'data_blob', 'commentary');
```

Comment:
Could you confirm why these specific fields are being sanitised and not other fields in the `project_progress` data blob including:

- delivery_stage
- adjustment_request_status
- delivery_rag
- project_status

Reply:
I mentioned in the other comment, I didn’t sanitise every field, which wasn't revealing a lot but these also can be replace with lorem_ipsum if want to.

Reply:
Cool, understood - let's just sanitise all of the fields in these tables for consistency - sanitisation by default, unless there is a special reason not to